src/main/java/io/github/thebusybiscuit/slimefun4/core/commands/SlimefunTabCompleter.java

```diff
@@ -71,14 +76,14 @@ else if (args[0].equalsIgnoreCase("research")) {
         List<String> returnList = new ArrayList<>();
 
         for (String item : list) {
-            if (item.contains(input)) {
+            if (item.startsWith(input)) {
```

Comment:
Changing this to .startsWith() makes no sense here, especially because it would suddenly exclude a ton of results.

Reply:
IMO it adds unnecessary items that aren't gonna be used anyways (ex typing in "du" for "duct tape" will drown it in anything containing "dust"). I'll change it back if you prefer it with contains.